publish.sh

```diff
@@ -45,6 +45,8 @@ tag_n_push() {
 }
 
 tag_n_push "${PATCH}"
-tag_n_push "${MINOR}"
-tag_n_push "${MAJOR}"
-tag_n_push "latest"
+if echo "${PATCH}" | grep -qvP '^v?\d+\.\d+\.\d+-rc\d*$' ; then
```

Comment:
I think it might be clearer to test $VERSION instead but this should be fine for now.